### PR TITLE
Today I will cause problems on purpose: Clown Canegun

### PR DESCRIPTION
**Pull request by karmaisblackandbluepilled**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
add: added clown shotgun, disguised as a pimpstick, from old hippie and added it to the uplink
/:cl:

## About The Pull Request
*Redacted because this is inappropriate for GitHub*

## Why It's Good For The Game
because it's fun

### CODE REVIEW

**Comment by cris5311** [comment]
+1
clown best

File: hippiestation/code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -61,6 +62,13 @@
 	cost = 18
 	item = /obj/item/seeds/gatfruit/syndi
 
+/datum/uplink_item/role_restricted/canegun
+	name = "Concealed cane shotgun"
+	desc = "A shotgun cleverly disgusied as a pimp stick. Pull on it to rack it and fold it to fire. Holds 8 shells at once. Keep away from mimes."
+	restricted_roles = list("Clown")
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/uplink/uplink_items.dm]: Missing Mime, it was available for him too.

Comment (karmaisblackandbluepilled) [reply hippiestation/code/modules/uplink/uplink_items.dm]: alright I'll fix it

**Comment by QuarianCommando** [comment]
+1 once it's fixed. Someone add the old hippie port tag, assuming we still have it.

**Comment by HideAndSeekLOGIC** [comment]
+1, fix vars

**Comment by karmaisblackandbluepilled** [comment]
which are they

**Comment by karmaisblackandbluepilled** [comment]
fug u travis this works well ingame, I tested it.

**Comment by yoyobatty** [comment]
Fix the merge conflict by fetching upstream and running `git merge upstream/master` if it conflicts, just delete the lines from between ====== and MASTER

**Pull request closed without merging**